### ISSUE TITLE
Error when using @nimiq/core-web

### ISSUE THREAD

**Issue by smdmitry** [nimiq/core-js#508]
I try to use @nimiq/core-web in app like:
`npm i @nimiq/core-web
import Nimiq from '@nimiq/core-web';`

But browser throws error:
`Uncaught (in promise) ReferenceError: GetTransactionsProofMessage is not defined
    at Module.eval (web.esm.js?6a96:1)
    at eval (web.esm.js?6a96:2)
    at Module../node_modules/@nimiq/core-web/web.esm.js (vendors~nimiq-191a06aaa720ddb02fda.js:11)
    at __webpack_require__ (build.js?v=1.5.3-11:770)
    at fn (build.js?v=1.5.3-11:130)`

Many other packages install and work ok.

**Comment by sisou** [comment]
Hello Dmitry,

I'll look into it. Can you please confirm if you are using v1.5.0 of the package?

**Comment by smdmitry** [comment]
Yest, latest version 1.5.0.
Im using Vue with vue-loader/lib/plugin and webpack 4.35.3 if this helps.

**Comment by smdmitry** [comment]
Your fixes in https://github.com/nimiq/core-js/pull/509 fixed this issue, thank!

There is next issue though. web.esm.js is not suitable for using in webworker.
It tries to load worker via document, which is not present in webworker.

I have implemented workaround in my app, but it'll be nice to have build for webworker usage. 

Also I found that some place modifies Array.prototype and adds method keys. And in now iteration over any array (for (i in array) {...}) will include index 'keys'.
